src/MAUI/Maui.Samples/Samples/Map/ManageBookmarks/ManageBookmarks.xaml

```diff
@@ -3,21 +3,23 @@
              xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
-    <Grid Style="{DynamicResource EsriSampleContainer}">
-        <esriUI:MapView x:Name="MyMapView" Style="{DynamicResource EsriSampleGeoView}" />
-        <Border MinimumWidthRequest="350" Style="{DynamicResource EsriSampleControlPanel}">
-            <HorizontalStackLayout Padding="5" Spacing="5">
-                <Label HorizontalTextAlignment="Center"
-                       Text="Bookmark:"
-                       VerticalTextAlignment="Center" />
-                <Picker x:Name="bookmarkPicker"
-                        SelectedIndexChanged="BookmarkPicker_SelectedIndexChanged"
-                        VerticalOptions="CenterAndExpand"
-                        WidthRequest="150" />
-                <Button x:Name="ButtonAddBookmark"
-                        Clicked="ButtonAddBookmark_Clicked"
-                        Text="New" />
-            </HorizontalStackLayout>
-        </Border>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <HorizontalStackLayout Padding="5" Spacing="5">
+            <Label HorizontalTextAlignment="Center"
+                   Text="Bookmark:"
+                   VerticalTextAlignment="Center" />
+            <Picker x:Name="bookmarkPicker"
+                    SelectedIndexChanged="BookmarkPicker_SelectedIndexChanged"
+                    VerticalOptions="CenterAndExpand"
+                    WidthRequest="200" />
+            <Button x:Name="ButtonAddBookmark"
```

Comment:
Nitpick: feels like there might be slightly too much height on this button or not enough of a vertical margin.